src/lib/y2security/clients/security_policy_proposal.rb

```diff
@@ -477,10 +478,10 @@ def initialize(rule, links_builder)
 
         # @return [String]
         def to_html
-          return message if actions.none?
+          second_line = (rule.identifiers + rule.references).join(", ")
```

Comment:
Just a note for further discussion (not to block this PR): thinking about translations, should we programmatically insert a comma?